pkg/storage/sqlcommon/sqlcommon.go

```diff
@@ -150,6 +150,12 @@ type ContToken struct {
 	ObjectType string `json:"ObjectType"`
 }
 
+// ContToken represents a continuation token structure used in pagination.
+type ContTokenLegacy struct {
```

Comment:
Why is this called `Legacy`

Reply:
Can't find any usage for it either 🤔

Reply:
ah, crap, this wasn't supposed to get in